website/docs/r/is_vpn_server_client.html.markdown

```diff
@@ -23,8 +23,8 @@ resource "ibm_is_vpn_server_client" "example" {
 ## Argument Reference
 Review the argument references that you can specify for your resource. 
 
-- `vpn_server` - (Required, Forces new resource, String) The VPN server identifier.
-- `vpn_client` - (Required, Forces new resource, String) The VPN client identifier.
+- `vpn_server` - (Required, String) The VPN server identifier.
+- `vpn_client` - (Required, String) The VPN client identifier.
```

Comment:
i see these having forcenew true. are we planning to remove it ?

Reply:
added again